gnomad_qc/analyses/compute_pext.py

```diff
@@ -109,7 +137,7 @@ def get_pipeline_resources(
         pipeline_name="compute_pext",
         pipeline_resources={
             "context HT": {"context_ht": gnomad_res.vep_context.versions[vep_version]},
-            "GTEx RSEM MT": {"tx_mt": gnomad_res.gtex_rsem.versions[gtex_version]},
+            "GTEx RSEM MT": {"tx_mt": tx_mt},
```

Comment:
maybe add a NOTE (or TODO) here that you can flip this line back to `"GTEx RSEM MT": {"tx_mt": gnomad_res.gtex_rsem.versions[gtex_version]},` once the public version is available? just since that is obvious now but may be forgotten later